src/components/OrderableList/OrderableList.stories.tsx

```diff
@@ -0,0 +1,250 @@
+import { Button, ButtonSize } from "@components/Button";
+import { OrderableList as OrderableListComponent } from "./OrderableList";
+import { merge } from "@utilities/merge";
+import { GridNode } from "@react-types/grid";
+import { DragProperties, OrderableListItem } from "./types";
+import { FocusController, FocusControllerWidth, ItemDragState, OrderableListProps } from ".";
+import { chain } from "@react-aria/utils";
+import { Checklist, ChecklistDirection } from "@components/Checklist";
+import { Textarea } from "@components/Textarea";
+import { ButtonGroup } from "@components/ButtonGroup";
+
+// eslint-disable-next-line import/no-default-export
+export default {
+    title: "Components/Orderable List",
+    component: OrderableListComponent,
+    args: {
+        dragDisabled: false,
+        disableTypeAhead: true,
+    },
+    argTypes: {
+        onMove: { action: "onMove" },
+    },
+} as Meta<OrderableListProps<StoryListItem>>;
+
+export const dragStoryStyles: Record<ItemDragState, string> = {
+    [ItemDragState.Dragging]: "tw-bg-black-10 tw-border-black-20 tw-opacity-75",
+    [ItemDragState.Idle]: "tw-border-black-20",
+    [ItemDragState.Preview]: "tw-bg-white tw-border-violet-70 tw-border-4",
+};
+
+export enum HighlightColor {
+    Violet = "Violet",
+    Green = "Green",
+    Red = "Red",
+}
+
+export const HighlightClasses: Record<HighlightColor, string> = {
+    [HighlightColor.Violet]: "tw-text-violet-60",
+    [HighlightColor.Green]: "tw-text-green-60",
+    [HighlightColor.Red]: "tw-text-red-60",
+};
+
+export type HighlightProps = {
+    color: HighlightColor;
+    children: ReactChild;
+};
+
+export type StoryListItem = {
+    content: JSX.Element;
+};
+
+const renderContent = (
+    { value, prevKey, nextKey }: GridNode<OrderableListItem<StoryListItem>>,
+    { componentDragState, isFocusVisible }: DragProperties,
+) => {
+    let position = "Middle";
+    if (!prevKey) {
+        position = "First";
+    } else if (!nextKey) {
+        position = "Last";
+    }
+    return (
+        <div
+            className={merge([
+                "tw-break-word tw-border tw-border-black-40 tw-border-solid tw-p-3",
+                dragStoryStyles[componentDragState],
+                isFocusVisible && "tw-bg-violet-20",
+            ])}
+        >
+            <div className="tw-text-xs tw-text-black-60">Position: {position}</div>
+            <hr className="tw-mt-2 tw-mb-2 tw-border-black-20 tw-bg-black-20" />
+            {value.content}
+            <hr className="tw-mt-3 tw-mb-2 tw-border-black-20 tw-bg-black-20" />
+            <div className="tw-flex tw-justify-between tw-text-s">
+                <span className="tw-font-medium">{isFocusVisible && "Im in keyboard focus"}</span>
+                <span>Drag State: {componentDragState}</span>
+            </div>
+        </div>
+    );
+};
+
+const ToggleableChecklist = () => {
+    const [checked, setChecked] = useState<string[]>([]);
+    return (
```

Comment:
When some checkboxes are active and the Item then gets reordered, all boxes are unchecked

Reply:
fixed :)